pages/index.tsx

```diff
@@ -37,6 +41,13 @@ export const HomePage: NextPage = () => {
   const { data: allNeeds } = useQuery('getAllNeeds', () => getAllNeeds())
   const [selectedNeeds, setSelectedNeeds] = useState<string[]>([])
 
+  const { data: allNeighborhoods } = useQuery('selectAllNeighborhoods', () =>
+    selectAllNeighborhoods()
+  )
+  const [selectedNeighborhoods, setSelectedNeighborhoods] = useState<
+    Neighborhood[]
+  >([])
```

Comment:
Instead of making the `selectedNeighborhoods` the a list of whole `Neighborhood` objects, maybe we make it just a list of neighborhood ids? I think it would make the code a bit simpler.

Reply:
Ack, the reason this prob isn't simpler is my fault - the `Resource` type doesn't have all the fields the Airtable does because we just don't use all of them and there are a lot so I just didn't add all of them 😬 

You should be able to add a `Neighborhoods` field and use that to filter:

```
export type Resource = {
    id: string
    ...
    /** Single neighbhorhood table id - array shape comes from Airtable mapping */
    Neighborhoods?: string[]
    ...
}
```